modules/using-must-gather.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/modules/troubleshooting.adoc
+// * migration/migrating_4_1_4/modules/troubleshooting.adoc
+// * migration/migrating_4_2_4/modules/troubleshooting.adoc
+
+[id="using-must-gather_{context}"]
+= Gathering data for a customer support case
+
+If you open a customer support case, you can run the `must-gather` tool with the `openshift-migration-must-gather-rhel8` image to collect information about your cluster and upload it to the link:https://access.redhat.com[Red Hat Customer Portal]. The `openshift-migration-must-gather-rhel8` image collects logs and Custom Resource data that are not collected by the default `must-gather` image.
```

Comment:
I think it might be a good idea to put a paragraph break between the two sentences, mainly because the sentences are long. Easier to scan the meaning if there is a break.

Reply:
Fixed.